src/urllib3/connection.py

```diff
@@ -160,22 +211,28 @@ def _new_conn(self):
 
```

Comment:
Line 210 - `NewConnectionError` expects a `ConnectionPool` - should it then be raised by `ConnectionPool` or this exception should be modified?